dice_ml/data_interfaces/private_data_interface.py

```diff
@@ -351,8 +351,12 @@ def prepare_df_for_ohe_encoding(self):
 
         return df
 
-    def prepare_query_instance(self, query_instance):
-        """Prepares user defined test input(s) for DiCE."""
+    def ensure_consistent_type(self, output_df, query_instance):
```

Comment:
Could you defined this function in https://github.com/interpretml/DiCE/blob/main/dice_ml/data_interfaces/base_data_interface.py? This will be inherited in both public and private data.

Reply:
oh yes, good idea. Thank you, will do this.